src/algebra/jordan/special.lean

```diff
@@ -0,0 +1,71 @@
+/-
+Copyright (c) 2021 Christopher Hoskin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christopher Hoskin
+-/
+import algebra.symmetrized
+import algebra.jordan.basic
+
+/-!
+# Special Jordan algebras
+
+A commutative multiplication on a real or complex space can be constructed from any multiplication
+by "symmetrisation" i.e
+```
+a∘b = 1/2(ab+ba).
+```
+When the original multiplication is associative, the symmetrised algebra is a commutative Jordan
+algebra. A commutative Jordan algebra which can be constructed in this way from an associative
+multiplication is said to be a special Jordan algebra.
+
+## Main results
+
+- `is_comm_jordan` : The symmeterised algebra arising from an associative algebra is a commutative
+  Jordan algebra.
+
+## References
+
+* [Hanche-Olsen and Størmer, Jordan Operator Algebras][hancheolsenstormer1984]
+-/
+
+
+open sym_alg
+
+/- 2 commutes with every element of a ring -/
+lemma two_commute {α : Type*} [ring α] (a : α) : commute 2 a := begin
+  --convert commute.semiconj_by 2 a,
+  unfold _root_.commute,
+  rw [semiconj_by, mul_two, two_mul],
+end
+
+/- If 2 is invertible, ⅟2 commutes with every element of a ring -/
+lemma half_commute {α : Type*} [ring α] [invertible (2 : α)] (a : α) : commute (⅟2) a :=
+  commute.inv_of_left (two_commute a)
+
+/- The symmetrisation of a real (unital, associative) algebra multiplication is a commutative
+Jordan non-associative ring -/
+instance (α : Type*) [ring α] [invertible (2 : α)] : is_comm_jordan (αˢʸᵐ) :=
+{ mul_comm := sym_alg.mul_comm,
+  jordan := λ a,
+  begin
+    intro,
+    change ⅟2 * (⅟2 * (unsym a * unsym b + unsym b * unsym a) * unsym(a*a)
+      + unsym(a*a) * (⅟2 * (unsym a * unsym b + unsym b * unsym a)))
+      = ⅟2 * (unsym a * (⅟2 * (unsym b * unsym(a*a) + unsym(a*a) * unsym b))
+        + (⅟2 * (unsym b * unsym(a*a) + unsym(a*a) * unsym b)) * unsym a),
```

Comment:
Instead of using `change`, can you use the lemmas we added in the other PR?

Reply:
Done :)